### ISSUE TITLE
Template Inheritance

### ISSUE THREAD

**Issue by paulyoung** [deedubs/require-jade#10]
Should this work?

**Comment by vincentmac** [comment]
It would be nice to have, but at this point I don't think it works.  I haven't tried it out, so I cannot verify that.

Can you do template inheritance in jrburke text module?

**Comment by paulyoung** [comment]
I don't think so. Template inheritance is a feature of Jade.

https://github.com/visionmedia/jade#a11

**Comment by vincentmac** [comment]
Yes. Inheritance works wonderfully on the server side. I've kept my frontend templates pretty simple so far. I wondered about inheritance too, but ended up using backbone to control smaller sub views. 

Have you asked TJ about this?

**Comment by Rush** [comment]
Would be REALLY nice to have this feature.
